app/main.py

```diff
@@ -2,0 +1,8 @@
+def copy_file(command: str) -> None:
+    cd = command.split(" ")
+    if cd[1] == cd[2]:
```

Comment:
The function does not validate the input to ensure it contains exactly three elements and starts with 'cp'. According to the checklist, you should check that the input string contains 3 elements separated by space, and has `cp` on the first place.